pkgs/development/python-modules/drawilleplot/default.nix

```
@@ -0,0 +1,21 @@
+{lib, buildPythonPackage, fetchPypi, drawille, matplotlib}:
+
+buildPythonPackage rec {
+  pname = "drawilleplot";
+  version = "0.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-ZEDroo7KkI2VxdESb2QDX+dPY4UahuuK9L0EddrxJjQ=";
+  };
+
+  propagatedBuildInputs = [drawille matplotlib];
```

```suggestion
  propagatedBuildInputs = [
    drawille
    matplotlib
  ];
```

Expand lists to one item per line.